src/main/resources/hibernate.cfg.xml

```diff
@@ -2,7 +2,7 @@
     <session-factory>
         <property name="dialect">org.hibernate.dialect.MySQL5Dialect</property>
         <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app?
-            serverTimezone=UTC</property>
+serverTimezone=UTC</property>
```

Comment:
I meant that we should better write 1 property per line
100 character limit can be omitted in configurational files

so we can write this property on 1 line even if it is a bit longer than 100 characters - this will look better